changelog.d/20241004_101450_rchard_set_default_max_workers.rst

```diff
@@ -0,0 +1,34 @@
+.. A new scriv changelog fragment.
+..
+.. Uncomment the header that is right (remove the leading dots).
+..
+.. New Functionality
+.. ^^^^^^^^^^^^^^^^^
+..
+.. - A bullet item for the New Functionality category.
+..
+.. Bug Fixes
+.. ^^^^^^^^^
+..
+.. - A bullet item for the Bug Fixes category.
+..
+.. Removed
+.. ^^^^^^^
+..
+.. - A bullet item for the Removed category.
+..
+.. Deprecated
+.. ^^^^^^^^^^
+..
+.. - A bullet item for the Deprecated category.
+..
+Changed
+^^^^^^^
+..
+- Set max_workers_per_node=1 in the default LocalProvider config.
```

Comment:
Consider this version:
```suggestion
- Set ``max_workers_per_node=1`` in the default endpoint config.
```

Reply:
I also suggest that we add some context for _why_.  As in only ~half of folks (if that) will immediately "get" the backstory.  Perhaps something like:

```restructuredText
- Prevent unintended hogging of resources (e.g., login nodes) by setting the default
  endpoint configuration (which uses `LocalProvider`_) to only use a single worker
  (``max_workers_per_node=1``)

.. _LocalProvider: https://parsl.readthedocs.io/en/stable/stubs/parsl.providers.LocalProvider.html
```